TwilioVerifyDemo/TwilioVerifyDemo.xcodeproj/project.pbxproj

```diff
@@ -613,7 +613,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_ENTITLEMENTS = TwilioVerifyDemo/TwilioVerifyDemo.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Distribution";
```

Comment:
Can we keep distribution? This way developers won't need to use sandbox for push

Reply:
Done